### PR TITLE
[MI-3775]: Added test cases for reactions on MS Teams sync plugin.

### PR DESCRIPTION
**Pull request by arush-vashishtha**
**Summary**
This PR consists test cases for reactions in MS Teams sync plugin with following scenarios:

- Reaction to a message.
- Remove reaction.
- Change reaction.
- Multiple reaction by single and multiple user.
- Reaction from disconnected account.

### CODE REVIEW

File: data/test-cases/plugins/ms-teams-sync/user-message/Add_reaction_to_message.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+# (Required) Ensure all values are filled up
+name: "Add reaction to a message"
+status: Active
+priority: Normal
+folder: user-message
```
Comment (AayushChaudhary0001) [inline data/test-cases/plugins/ms-teams-sync/user-message/Add_reaction_to_message.md:6]: "user-reactions"

Comment (AayushChaudhary0001) [reply data/test-cases/plugins/ms-teams-sync/user-message/Add_reaction_to_message.md:6]: Fix it everywhere

**Review by AayushChaudhary0001 — APPROVED** [review]

File: data/test-cases/plugins/ms-teams-sync/user-reactions/Multiple_reactions_to_message.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+**Step 4**
+
+1. Add multiple reactions on a message from multiple users in the linked channel or GM on MS Teams.
+2. Navigate to the linked channel or GM on MM.
+
+**Expected**
+
+The user should be able to view the last added reaction on the message in the linked channel or DM/GM on MS Teams.
+After step 2, the user should be able to view the last added reaction on the message in the linked channel or DM/GM on MM.
+After step 3, the user should be able to view all the added reactions of different users on the message in the linked channel or GM on MS Teams.
```
Comment (ayusht2810) [inline data/test-cases/plugins/ms-teams-sync/user-reactions/Multiple_reactions_to_message.md:59]: is this correct? shouldn't we see the last reaction added by a particular user?

File: data/test-cases/plugins/ms-teams-sync/user-reactions/Remove_reaction_to_message.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+2. Navigate to the linked channel or DM/GM on MS Teams. 
+
+**Step 2**
+
+1. Remove reaction from a message in the linked channel or DM/GM on MS Teams.
+2. Navigate to the linked channel or DM/GM on MM.
+
+**Expected**
+
+The reaction should get removed from the message in the linked channel or DM/GM on MS Teams.
```
Comment (ayusht2810) [inline data/test-cases/plugins/ms-teams-sync/user-reactions/Remove_reaction_to_message.md:47]:
```suggestion
The reaction should be removed from the message in the linked channel or DM/GM on MS Teams.
```

File: data/test-cases/plugins/ms-teams-sync/user-reactions/Multiple_reactions_to_message.md (hunk 2 above)
Comment (arush-vashishtha) [reply data/test-cases/plugins/ms-teams-sync/user-reactions/Multiple_reactions_to_message.md:59]: it should show the last added added reaction by each distinguished user.

File: data/test-cases/plugins/ms-teams-sync/user-reactions/Disconnected_user_reactions.md [hunk 4]
```diff
@@ -39,10 +39,16 @@ steps_hashed: null
...
+1. Add reaction on a message in the linked channel on MM, when the account is disconnected and the bot account is connected.
+2. Navigate to the linked channel on MS Teams.
+
 **Expected**
 
 The reaction should not get added to the message in the linked channel on MM.
-After step 2, the reaction should get added to the message in the linked channel or DM/GM on MM.
\ No newline at end of file
+After step 2, the reaction should get added to the message in the linked channel or DM/GM on MM.
+After step 3, the reaction should not get added to the message in the linked channel on MS Teams.
```
Comment (ayusht2810) [inline data/test-cases/plugins/ms-teams-sync/user-reactions/Disconnected_user_reactions.md:54]:
```suggestion
After step 3, the reaction should not get added to the message in the linked channel or DM/GM on MS Teams.
```

**Review by ayusht2810 — APPROVED** [review]

**Pull request merged**